assets/wizards/popups/views/campaigns/index.js

```diff
@@ -0,0 +1,301 @@
+						options={ [
+							{ key: 'active', name: __( 'Active Prompts', 'newspack' ) },
+							{ key: 'unassigned', name: __( 'Unassigned Prompts', 'newspack' ) },
+							{
+								key: 'header-campaigns',
+								name: __( 'Campaigns', 'newspack' ),
+								className: 'is-header',
+							},
+							...activeCampaigns.map( ( { term_id: id, name } ) => ( {
+								key: id,
+								name,
+								className: 'newspack-campaigns__campaign-group__select-control-group-item',
+							} ) ),
+							archivedCampaigns.length && {
+								key: 'header-archived-campaigns',
+								name: __( 'Archived Campaigns', 'newspack' ),
+								className: 'is-header',
+							},
+							...archivedCampaigns.map( ( { term_id: id, name } ) => ( {
+								key: id,
+								name,
+								className: 'newspack-campaigns__campaign-group__select-control-group-item',
+							} ) ),
+						] }
+						onChange={ ( { selectedItem: { key } } ) =>
+							'active' === key
+								? history.push( '/campaigns' )
+								: history.push( `/campaigns/${ key }` )
+						}
+						value={ valueForCampaignId( campaignId ) }
+						hideLabelFromVision={ true }
+					/>
+					{ campaignData && (
+						<div className="newspack-campaigns__campaign-group__filter-group-actions__button">
+							<Button
+								isQuaternary
+								isSmall
+								className={ popoverVisible && 'popover-active' }
+								onClick={ () => setPopoverVisible( ! popoverVisible ) }
+								icon={ moreVertical }
+								label={ __( 'Actions', 'newspack' ) }
+							/>
+							{ popoverVisible && (
+								<CampaignManagementPopover
+									dismiss={ () => setPopoverVisible( false ) }
+									isArchive={ campaignData && 'archive' === campaignData.status }
+									onActivate={ () => manageCampaignGroup( prompts ) }
+									onArchive={ () => archiveCampaignGroup( campaignId, true ) }
+									onDeactivate={ () => manageCampaignGroup( prompts, 'DELETE' ) }
+									onDelete={ () => deleteCampaignGroup( campaignId ) }
+									onDuplicate={ () => {
+										setModalVisible( true );
+										setCampaignName( '' );
+										setModalType( MODAL_TYPE_DUPLICATE );
+									} }
+									onRename={ () => {
+										setCampaignName( campaignData.name );
+										setModalVisible( true );
+										setModalType( MODAL_TYPE_RENAME );
+									} }
+									onUnarchive={ () => archiveCampaignGroup( campaignId, false ) }
+									hasUnpublished={ prompts.some( ( { status } ) => 'publish' !== status ) }
+									hasPublished={ prompts.some( ( { status } ) => 'publish' === status ) }
+								/>
+							) }
+						</div>
+					) }
+				</div>
+				<div className="newspack-campaigns__campaign-group__add-new-button">
+					<Button
+						isPrimary
+						isSmall
+						onClick={ () => {
+							setModalVisible( ! modalVisible );
+							setCampaignName( '' );
+							setModalType( MODAL_TYPE_NEW );
+						} }
+					>
+						{ __( 'Add New Campaign', 'newspack' ) }
```

Comment:
Suggestion: some space could be saved by moving this button to segment header:

![image](https://user-images.githubusercontent.com/7383192/106473153-be4b8800-64a3-11eb-8244-4b0087f0f85a.png)

Reply:
I made this change in https://github.com/Automattic/newspack-plugin/pull/833/commits/01cd2fdd52c3b4de2c9ec72277a0580146300e5a, but curious what @thomasguillot thinks. If you don't like it, I can revert it easily.

Reply:
@jeffersonrabb, I don't like it.

Reply:
Per @thomasguillot's UI feedback I've reverted the change in https://github.com/Automattic/newspack-plugin/pull/833/commits/ecfa5d1618edc117a87b50800083fde2971684ae